RHEL/5/input/xccdf/system/auditing.xml

```diff
@@ -447,23 +445,26 @@ to the system time should be audited.</rationale>
 
 <Rule id="audit_rules_time_stime">
 <title>Record Attempts to Alter Time Through stime</title>
-<description>On a 32-bit system, add the following to <tt>/etc/audit/audit.rules</tt>:
+<description>Add the following line to <tt>/etc/audit/audit.rules</tt> for both
+32-bit and 64-bit systems:
 <pre># audit_time_rules
--a exit,always -F arch=b32 -S stime -k audit_time_rules</pre>
-On a 64-bit system, the "-S stime" is not necessary. The -k option allows for 
-the specification of a key in string form that can be used for better 
-reporting capability through ausearch and aureport. Multiple system calls 
-can be defined on the same line to save space if desired, but is not required.
-See an example of multiple combined syscalls:
-<pre>-a exit,always -F arch=b64 -S adjtimex -S settimeofday -S clock_settime 
--k audit_time_rules</pre>
+-a always,exit -F arch=b32 -S stime -k audit_time_rules</pre>
+Since the 64-bit version of the "stime" system call is not defined in the audit
+lookup table, the corresponding "-F arch=b64" form of this rule is not expected
+to be defined on 64-bit systems (the aforementioned "-F arch=b32" stime rule
+form itself is sufficient for both 32-bit and 64-bit systems). The -k option
+allows for the specification of a key in string form that can be used for
+better reporting capability through ausearch and aureport. Multiple system
+calls can be defined on the same line to save space if desired, but is not
+required. See an example of multiple combined syscalls:
+<pre>-a always,exit -F arch=b64 -S adjtimex -S settimeofday -k audit_time_rules</pre>
```

Comment:
@iankko since stime is 32-bit wouldn't you want to change the example here to be `arch=b32` and not `arch=b64` in Fedora and RHEL instances of this rule?